bazel/ngraph.BUILD

```diff
@@ -314,17 +270,20 @@ cc_binary(
         "-fstack-protector-all",
         '-D SHARED_LIB_PREFIX=\\"lib\\"',
         '-D SHARED_LIB_SUFFIX=\\".so\\"',
-        '-D NGRAPH_VERSION=\\"0.19.0-rc.4\\"',
+        '-D NGRAPH_VERSION=\\"0.25.1-rc.0\\"',
```

Comment:
We should document that these ngraph versions need to change in this checklist: https://github.com/tensorflow/ngraph-bridge/wiki/Release-Checklist

Reply:
Already did.